contracts/test/walletAction-test.ts

```diff
@@ -176,6 +180,56 @@ describe("WalletActions", async function () {
     );
   });
 
+  it("should send ETH with function call via fallback expander", async function () {
+    // send money to sender bls wallet
+    const sendWallet = await fx.lazyBlsWallets[0]();
+    const ethToTransfer = parseEther("0.001");
+    await fx.signers[0].sendTransaction({
+      to: sendWallet.address,
+      value: ethToTransfer,
+    });
+
+    const MockAuction = await ethers.getContractFactory("MockAuction");
+    const mockAuction = await MockAuction.deploy();
+    await mockAuction.deployed();
+
+    expect(await fx.provider.getBalance(sendWallet.address)).to.equal(
+      ethToTransfer,
+    );
+    expect(await fx.provider.getBalance(mockAuction.address)).to.equal(0);
```

Comment:
https://www.chaijs.com/plugins/chai-as-promised/

```suggestion
    await expect(fx.provider.getBalance(sendWallet.address)).to.eventually.equal(
      ethToTransfer,
    );
    await expect(fx.provider.getBalance(mockAuction.address)).to.eventually.equal(0);
```

Reply:
Love it. Let's do it everywhere: https://github.com/web3well/bls-wallet/pull/518.